checks/signed_releases.go

```diff
@@ -18,19 +18,22 @@ import (
 	"strings"
 
 	"github.com/google/go-github/v32/github"
-	"github.com/ossf/scorecard/checker"
+	"github.com/ossf/scorecard/lib"
 )
 
-var releaseLookBack int = 5
+const (
+	signedReleasesStr = "Signed-Releases"
+	releaseLookBack   = 5
```

Comment:
Since you are refactoring things, how about doing s/releaseLookBack/releaseLoockBackDays. some of these vars will be more clear with what they represent.

Reply:
Done.